system/core/base.php

```diff
@@ -528,6 +528,7 @@ function hypha_setPage($page, $language, $name, $private) {
 				$page->appendChild($newLanguage);
 			}
 		}
+		$private = in_array($private, ['true', '1', 'on']) ? 'on' : 'off';
```

Comment:
Is this "is private" check (or rather, "normalize boolean" check) something to put in a util function (which should probably return bool, not 'on'/'off'? Probably something that is fine for a new commit after merging this PR, though.

Reply:
I'm not really sure what kind of improvements this commit really makes. Ideally, the commit message should explain any change in behaviour, just saying "improved" is vague unless it is really obvious from a very short commit (and even then, explaining is better).

Reply:
Agreed, a util function would be better.

I've updated the commit message to explain the change.